application.example.yaml

```diff
@@ -54,11 +54,14 @@ notifications:
 
 trading:
   # The percentage difference between the "long" and "short" exchanges before we will open our positions.
-  entrySpread: 0.0080
+  # Fees are added to this amount
+  # Increasing this value will make trades harder to enter
+  entrySpread: 0.0019
 
-  # The percentage difference *in profit only* before we will close our positions.
-  # Fees will be added to this amount.
-  exitTarget: 0.0050
+  # The percentage difference between the "long" and "short" exchanges before we will open our positions.
```

Comment:
I guess you mean `close our positions` instead of `open our positions`?

Reply:
Once this comment is fixed I think we can merge. Since it's a breaking change I'm contemplating a version bump to v0.12.0 in this PR as well though. I can fix the milestones to match.

Reply:
I don't think it is worth to merge this breaking change, knowing that we want to go toward a dynamic entry target and a minimum profit. let me know what you think

Reply:
That sounds fine to me. The settings work as they are even if they’re a bit confusing, and I would like to eliminate them altogether relatively soon. I’ll go ahead and close the PR.